src/sap.m/src/sap/m/ListItemBase.js

```diff
@@ -49,6 +49,11 @@ sap.ui.define(['jquery.sap.global', './library', 'sap/ui/core/Control', 'sap/ui/
 			unread : {type : "boolean", group : "Misc", defaultValue : false},
 
 			/**
+			 * Whether the control should be enabled. If set to false, the item is rendered as disabled.
+			 */
+			enabled : {type : "boolean", group : "Appearance", defaultValue : true},
```

Comment:
@StErMi FOL, thank you very much for your pull request. You have mentioned in the API "enabled" property but it means more than only selectability. When I see your implementation I would say the property should be "selectable" . You should think about adding the @since tag into the JSDoc.

Also you need to consider the followings

- Keyboard handling: You should have a look ListItemBase#onsapspace (Simply ListItemBase#isSelectable should return the new variable value then I believe most of the stuff should work fine)
- Accessibility: We may need to announce when the item is not selectable.
- As you probably know the rows(ColumnListItem) of the sap.m.Table inherits from ListItemBase and Table has (de)selectall checkbox for MultiSelect mode. If all rows are not selectable then we may need to update this checkbox as well(disabled).
- GroupHeaderListItems are not selectable by default (for now). We need to document this as well.
- We need some test pages. Probably in explored app.
- The most important think is the unit tests. We cannot deliver any feature without unit test also above cases can be capt by functional tests easily.